runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -6874,7 +6879,7 @@ void acquire_comm_dom_and_req_buf(c_nodeid_t remote_locale, int* p_rbi)
 
   want_cdi = want_cdi_start = comm_dom_free_idx;
   want_cd = &comm_doms[want_cdi];
-
+  int spins = 0;
```

Comment:
Just for completeness because if I didn't comment on this one you might wonder why, I believe this spin-wait will probably not be hurt if multiple QSBR checkpoints occur during the wait because here the performance is driven primarily by the rate at which remote nodes consume executeOn AM operations initiated by this node, and to the extent QSBR might affect that I've commented elsewhere.